internal/reports/report_v2.go

```diff
@@ -106,7 +106,7 @@ type StructuredReport struct {
 	NameChanges          []NameChange         `json:"nameChanges"`
 	TableReports         []TableReport        `json:"tableReports"`
 	UnexpectedConditions UnexpectedConditions `json:"unexpectedConditions"`
-	SchemaOnly           bool                 `json:"-"`
+	SchemaOnly           bool                 `json:"schemaOnly"`
```

Comment:
Do we need this deserialized in the report?

Reply:
> 

Not needed here, removed it!